packages/lib-classifier/src/store/FeedbackStore.js

```diff
@@ -23,6 +22,13 @@ const FeedbackStore = types
       return flatten(Array.from(self.rules.values()))
         .some(rule => rule.hideSubjectViewer)
     },
+    get isActive () {
+      const { projects = {}, subjects = {}, workflows = {} } = getRoot(self)
+      return helpers.isFeedbackActive(projects.active, subjects.active, workflows.active)
+    },
+    get isValid () {
```

Comment:
As discussed in person:

- I'm not in love with the naming, but I understand why now
- More importantly, if this is a workaround for a bug, we need a comment to that effect and a link to an issue

Reply:
No problem. This is a workaround for #1112 (feedback incorrectly being active when there are no rules.)

Reply:
![](https://media.giphy.com/media/CrnB9nnbVtey4/giphy.gif)

Reply:
With this PR, a subject can't have `isActive` true and an empty rules array. In theory (and according to the tests.)